tests/test_disco.py

```diff
@@ -0,0 +1,26 @@
+import json
+from pathlib import Path
+from unittest.mock import patch
+
+import pytest
+from tests.test_utils import TESTS_DIR
+
+from averell.readers.disco import get_features
+from averell.readers.disco import parse_xml
+
+
+@pytest.fixture
+def disco():
+    return json.loads(Path("tests/fixtures/disco.json").read_text())
```

Comment:
Try to avoid `tests/fixtures/disco.json` and use a proper Path() instead for win/unix compatibility

Reply:
Done